server/migrations/versions/f152c9a38faa_.py

```diff
@@ -0,0 +1,106 @@
+"""empty message
+
+Revision ID: f152c9a38faa
+Revises: 5be6d1238476
+Create Date: 2022-03-21 05:51:31.711074
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'f152c9a38faa'
+down_revision = '5be6d1238476'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('author',
```

Comment:
I'm unsure if these are the correct commands that should run. The table already exists so we shouldn't be creating a new one. Perhaps that's why we had to do stamp head. How did you run the migration command? Did you run it on an existing db with the schema from the master branch?